.github/workflows/code-quality.yml

```
@@ -1,17 +1,36 @@
+    branches:
+      - main
+      - dev
+  workflow_call:
+
 jobs:
-  quality:
+  linting:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/setup
```

actionlint (1.7.7)

name is required in action metadata "/home/jailuser/git/.github/actions/setup/action.yml"

(action)